src/enable_variable_ops/ngraph_enter_in_catalog.cc

```diff
@@ -66,6 +66,47 @@ Status EnterInCatalog(Graph* graph, int graph_id) {
   GetReversePostOrder(*graph, &ordered);
 
   for (auto node : ordered) {
+    if (node->type_string() == "NGraphAssign") {
+      Node* input_1;
+      TF_RETURN_IF_ERROR(node->input_node(1, &input_1));
+      if (input_1->type_string() == "NGraphEncapsulate") {
+        NGRAPH_VLOG(4)
+            << "Input node type for NGraphAssign is NGraphEncapsulate";
+        // attach attribute _ngraph_remove to this NGraphAssign
+        node->AddAttr("_ngraph_remove", true);
+        // get variable shared_name
+        string shared_name;
+        TF_RETURN_IF_ERROR(GetSharedName(node, &shared_name));
+        // get attribute copy_to_tf
+        bool copy_to_tf;
+        TF_RETURN_IF_ERROR(
+            GetNodeAttr(node->attrs(), "copy_to_tf", &copy_to_tf));
+        // get attribute is_tf_just_looking
+        bool is_tf_just_looking;
+        TF_RETURN_IF_ERROR(GetNodeAttr(node->attrs(), "is_tf_just_looking",
+                                       &is_tf_just_looking));
+        // populate encap_output_info_map_
+        int output_index;
+        for (auto edge : node->in_edges()) {
```

Comment:
This might get you the edge
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/graph/graph.h#L194